34.md

```diff
@@ -181,6 +182,49 @@ The Status event with the largest created_at date is valid.
 
 The Status of a patch-revision defaults to either that of the root-patch, or `1632` (Closed) if the root-patch's Status is `1631` and the patch-revision isn't tagged in the `1631` event.
 
+## Git client nostr plugin (git-remote-nostr)
+
+Definition: Clone URL - a git remote URL listed under the `clone` tag of the repositories kind `30617` event
+
+The offical git client supports interacting with 'remotes' using custom protocols via [git-remote-helpers](https://git-scm.com/docs/gitremote-helpers).
+
+An implementation of git-remote-nostr SHOULD:
+
+### 1. Use Git Remote Nostr URL Format
+
+Struture:
+
+- `nostr://` - git syntax for a protocol URL
+- `<user>@` - optional ssh user for interacting with Clone URLs
```

Comment:
It can possibly be the NIP-05 user, too

Reply:
I guess overriding ssh user is not possible if using NIP-05 in the url.

Reply:
> It can possibly be the NIP-05 user, too

that is covered in line 202 and 203.